src/Symfony/Component/Console/Input/InputOption.php

```diff
@@ -25,6 +25,7 @@ class InputOption
     const VALUE_REQUIRED = 2;
     const VALUE_OPTIONAL = 4;
     const VALUE_IS_ARRAY = 8;
+    const VALUE_IS_COMMA_SEPARATED = 16;
```

Comment:
Which also have the benefit to make the `VALUE_IS_COMMA_SEPARATED_ARRAY` only available as a valid option including the `VALUE_IS_ARRAY`.